.npmignore

```diff
@@ -1,7 +1,8 @@
-**/*
-!README.md
-!LICENSE
-!PATENTS
-!package.json
-!typings/**/*
-!dist/**/*
\ No newline at end of file
+docs/
+src/
+test/
+theme/
+
+circle.yml
+ts*.*
```

Comment:
tslint, tsconfig since we don't ship ts source. it's not _essential_, just classy